src/consumption.cpp

```diff
@@ -1201,6 +1201,9 @@ bool player::feed_furnace_with( item &it )
     if( !can_feed_furnace_with( it ) ) {
         return false;
     }
+    if( it.is_favorite && !g->u.query_yn( _( "Are you sure you want to eat this favorited item?" ) ) ) {
```

Comment:
Would be nice to add the item's name to the message.
```suggestion
    if( it.is_favorite && !g->u.query_yn( _( "Are you sure you want to eat your favorited %s?" ), it.tname() ) ) {
```